packages/dataviews/src/components/dataviews/stories/index.story.tsx

```diff
@@ -130,44 +132,71 @@ function PlanetOverview( { planets }: { planets: SpaceObject[] } ) {
 	const moons = planets.reduce( ( sum, item ) => sum + item.satellites, 0 );
 
 	return (
-		<Card isBorderless style={ { padding: '12px 24px' } }>
-			<CardHeader>
-				<Heading level={ 2 }>{ __( 'Solar System numbers' ) }</Heading>
-				<DataViews.Search label={ __( 'moons by planet' ) } />
-			</CardHeader>
-
-			<CardBody>
-				<VStack spacing={ 2 }>
-					<Text size={ 18 } as="p">
-						{ createInterpolateElement(
-							_n(
-								'<PlanetsNumber /> planet',
-								'<PlanetsNumber /> planets',
-								planets.length
-							),
-							{
-								PlanetsNumber: (
-									<strong>{ planets.length } </strong>
-								),
-							}
-						) }
-					</Text>
-
-					<Text size={ 18 } as="p">
-						{ createInterpolateElement(
-							_n(
-								'<SatellitesNumber /> moon',
-								'<SatellitesNumber /> moons',
-								moons
-							),
-							{
-								SatellitesNumber: <strong>{ moons } </strong>,
-							}
-						) }
-					</Text>
-				</VStack>
-			</CardBody>
-		</Card>
+		<>
+			<Grid
+				templateColumns="repeat(auto-fit, minmax(330px, 1fr))"
+				justify="left"
+				align="flex-start"
+				className="dataviews-sticky-header"
+			>
+				<Card variant="secondary">
+					<CardHeader>
+						<Heading level={ 2 }>
+							{ __( 'Solar System numbers' ) }
+						</Heading>
+					</CardHeader>
+
+					<CardBody>
+						<VStack>
+							<Text size={ 18 } as="p">
+								{ createInterpolateElement(
+									_n(
+										'<PlanetsNumber /> planet',
+										'<PlanetsNumber /> planets',
+										planets.length
+									),
+									{
+										PlanetsNumber: (
+											<strong>{ planets.length } </strong>
+										),
+									}
+								) }
+							</Text>
+
+							<Text size={ 18 } as="p">
+								{ createInterpolateElement(
+									_n(
+										'<SatellitesNumber /> moon',
+										'<SatellitesNumber /> moons',
+										moons
+									),
+									{
+										SatellitesNumber: (
+											<strong>{ moons } </strong>
+										),
+									}
+								) }
+							</Text>
+						</VStack>
+					</CardBody>
+				</Card>
+
+				<HStack justify="end">
+					<DataViews.Search label={ __( 'moons by planet' ) } />
+				</HStack>
+
+				<HStack justify="end">
+					<DataViews.Pagination />
+					<DataViews.ViewConfig />
+					<DataViews.ViewTypeMenu defaultLayouts={ defaultLayouts } />
```

Comment:
`defaultLayouts` is a global. If this component were defined in its own file, it won't work.

Reply:
I've exposed the `defaultLayouts` through the Context. What do you think